jetstream/tests/engine/test_token_utils.py

```diff
@@ -48,7 +48,7 @@ def decode(self, t: int) -> str:
 
 class TokenUtilsTest(unittest.TestCase):
     def setup(self):
-        tokenizer_path = "tokenizer.model"
+        tokenizer_path = "third_party/llama2/tokenizer.model"
         current_dir = os.path.dirname(__file__)
         tokenizer_path = os.path.join(current_dir, tokenizer_path)
         print(f"model_path: {tokenizer_path}")
```

Comment:
Nit outside of this change: Should probably be renamed to tokenizer_path. 

Also typos in comments above "Tokenier" -> "Tokenizer"

Reply:
Thanks Pate! there could be more data set from llama2. Keep it as "third_party/llama2"